docs/docs/ref-02-component-api.md

```diff
@@ -124,11 +124,11 @@ Calling `setProps()` on a root-level component will change its properties and tr
 
 > Note:
 >
+> This method is deprecated and will be removed soon. Instead of calling `setProps`, try invoking ReactDOM.render() again with the new props. For additional notes, see our [blog post about using the Top Level API](/react/blog/2015/10/01/react-render-and-top-level-api.html)
```

Comment:
Can we leave in the piece that says it's not available on ES6 classes to avoid potential confusion?

> This method is deprecated and will be removed soon. It is not available on ES6 `class` components …